src/data/Data.jl

```diff
@@ -1,15 +1,34 @@
 module Data
 
-using CounterfactualExplanations
-using ..GenerativeModels
-using ..Models
 using Random
+using LazyArtifacts
```

Comment:
Also, the separation of these imports from the rest seems slightly odd to me - is there a principled reason for that?